spec.bs

```diff
@@ -2495,6 +2496,26 @@ container/fenced navigable=] will fail, as outlined in [[#navigation-patch]].
 
 </div>
 
+<h4 id=allow-automatic-beacons>The
+\`<a http-header><code>Allow-Fenced-Frame-Automatic-Beacons</code></a>\` HTTP response header</h4>
```

Comment:
(Maybe a fenced frame style thing, but not sure why there's a backtick surrounding `<a><code></a></code>`. I see it's already in the spec elsewhere though - so free to ignore me I guess)

Reply:
I see backticks for headers defined and referenced in other specs, so I think that's standardized.

[example from the Fetch spec](https://fetch.spec.whatwg.org/#http-access-control-allow-origin)
[example from PA spec](https://wicg.github.io/turtledove/#ad-auction-signals-header)
[example from HTML spec](https://html.spec.whatwg.org/multipage/browsers.html#origin-keyed-agent-clusters)

Reply:
I think it's just a personal preference thing - but like I said, feel free to ignore. :)